db/schema.rb

```diff
@@ -0,0 +1,42 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_09_11_125217) do
+
+  create_table "events", force: :cascade do |t|
+    t.datetime "event_date"
+    t.text "event_description"
+    t.string "event_location"
+    t.string "event_name"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "event_creator_id"
```

Comment:
The event table has creator_id as the foreign key. We are currently missing that.

Reply:
I don't understand what you mean. 
Currently, event_creator_id is the foreign key of events table
![image](https://user-images.githubusercontent.com/63403267/93062551-77233e80-f64b-11ea-82f7-fe7cff06ecc8.png)